cmd/analyze.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"errors"
+	"os"
+
+	"github.com/GoogleCloudPlatform/container-diff/differs"
+	"github.com/GoogleCloudPlatform/container-diff/utils"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+)
+
+var analyzeCmd = &cobra.Command{
+	Use:   "analyze",
+	Short: "Analyzes an image: [image]",
+	Long:  `Analyzes an image using the specifed analyzers as indicated via flags (see documentation for available ones).`,
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I'm assuming this is just a copy-paste from the analyze and diff functions?  So no need to do a full review of the code?